docs/install.md

```diff
@@ -335,35 +335,35 @@ Select the following tabs for the installation description of your target.
 
     <font size="+2">1. Generate a new Proxmox VM</font>
 
-    - Access the Proxmox web interface via HTTPS on TCP port **8006**:
+    1. Access the Proxmox web interface via HTTPS on TCP port **8006**:
         - URL: `https://<your.IP>:8006`
         - Username: `root`
         - Password: `<root user password you entered during Proxmox VE install>`
-    - Select the **Create VM** button at the top right corner. Choose a **Node** , a **VM ID** and a **Name**, then click **Next**.
+    1. Tab **General**: Select the **Create VM** button at the top right corner. Choose a **Node** , a **VM ID** and a **Name**, then click **Next**.
 
-        ![Proxmox VM creation](assets/images/proxmox2.png){: width="722" height="314" loading="lazy"}
+        ![Proxmox VM creation](assets/images/proxmox2.png){: width="600" height="260" loading="lazy"}
 
         Remember the VM ID, you need it later.
 
-    - Select "**Do not use any media** ", for the **Guest OS** assure that **Linux** and version "**5.x - 2.6 Kernel**" is selected, then click **Next**.
-    - As **Machine** you can select `q35`, but the older default will work as well. We recommend the para-virtualised **VirtIO SCSI** controller, which should be the default. Click **Next**.
-    - On the **Disks** tab, delete the default `scsi0` disk with the red trash bin button, then click **Next**.
-    - Adjust CPU details as required, we recommend to use the default `kvm64` type. Then click **Next**.
-    - While DietPi runs with less, depending on the software you want to install and run within the VM, we recommend at least 1024 MiB memory size, 2048 MiB allows the DietPi system to setup itself without a swap file by default. When done, click **Next**.
+    1. Tab **OS**: Select "**Do not use any media** ", for the **Guest OS** assure that **Linux** and version "**5.x - 2.6 Kernel**" is selected, then click **Next**.
+    1. Tab **System**: As **Machine** you can select `q35`, but the older default will work as well. We recommend the para-virtualised **VirtIO SCSI** controller, which should be the default. Click **Next**.
+    1. Tab **Disks**: Delete the default `scsi0` disk with the red trash bin button, then click **Next**.
+    1. Tab **CPU**: Adjust CPU details as required, we recommend to use the default `kvm64` type. Then click **Next**.
+    1. Tab **Memory**: While DietPi runs with less, depending on the software you want to install and run within the VM, we recommend at least 1024 MiB memory size, 2048 MiB allows the DietPi system to setup itself without a swap file by default. When done, click **Next**.
```

Comment:
Good idea to prefix it with the tab name 👍.

Numbering is incremented automatically? I hope it doesn't look confusing when the parent `<font size="+2">` numbering and this list numbering. I'll rebuild the test page so we can look at it.

Reply:
Looks fine, numbering works fine 👍.